src/packages/frontend/project/page/file-tab.tsx

```diff
@@ -514,6 +514,9 @@ function DisplayedLabel({ path, label, inline = true, project_id, condensed }) {
     const style: CSS = {
       fontSize: condensed ? "10px" : "12px",
       textAlign: "center",
+      maxWidth: "65px",
+      overflow: "hidden",
+      textOverflow: "ellipsis", // important for i18n, since sometimes the words are long
```

Comment:
This here is the only change with a visible difference for some languages (*cough* German *cough*). It limits the width of the labels in the vertical bar and adds dots. This doesn't change anything when using English.